templates/deployment.yaml

```diff
@@ -40,15 +40,15 @@ spec:
         {{- $agentImage := printf "%s:%s" .image.repository .image.tag }}
         - name: init-token-{{ .name }}
           securityContext:
-            capabilities:
-              add:
-              - IPC_LOCK
+            runAsUser: 1001
```

Comment:
what it is the reasoning behind this user?

Reply:
Indeed this is the postgres user probably been set here during a search/replace operation. Not relevant here. I will remove it